### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by ghost** [iXab3r/MicSwitch#32] [Show and tell]
Hi,

I just wanted to share the two icons that I made. They work really nicely on a dark windows task bar.

Richard

[Mic_Icons.zip](https://github.com/iXab3r/MicSwitch/files/6227678/Mic_Icons.zip)
![TaskBar](https://user-images.githubusercontent.com/80845164/112960286-8d569f00-9144-11eb-9fed-2b4912ce45d7.png)

**Comment by singlemalt8** [comment]
@rday74 - Thanks for sharing.  Would love to use these!  Can you explain where to put/rename the ico files to get MicSwitch to use them?  Do I somehow need to recompile?
